BloodManagmentSystem/packages.config

```diff
@@ -53,12 +50,18 @@
   <package id="Ninject.Web.Common.WebHost" version="3.2.0.0" targetFramework="net47" />
   <package id="Owin" version="1.0" targetFramework="net452" />
   <package id="Postal.Mvc5" version="1.2.0" targetFramework="net47" />
-  <package id="RazorEngine" version="3.4.1" targetFramework="net47" />
+  <package id="RazorEngine" version="3.10.0" targetFramework="net47" />
   <package id="Respond" version="1.4.2" targetFramework="net47" />
   <package id="RestSharp" version="105.2.3" targetFramework="net47" />
-  <package id="SendGrid.SmtpApi" version="1.1.3" targetFramework="net47" />
-  <package id="System.Diagnostics.DiagnosticSource" version="4.4.0" targetFramework="net47" />
-  <package id="System.Net.Http" version="4.0.0" targetFramework="net47" />
+  <package id="SendGrid.SmtpApi" version="1.3.3" targetFramework="net47" />
+  <package id="System.Diagnostics.DiagnosticSource" version="4.4.1" targetFramework="net47" />
+  <package id="System.IO" version="4.3.0" targetFramework="net47" />
+  <package id="System.Net.Http" version="4.3.3" targetFramework="net47" />
+  <package id="System.Runtime" version="4.3.0" targetFramework="net47" />
+  <package id="System.Security.Cryptography.Algorithms" version="4.3.0" targetFramework="net47" />
```

Comment:
Where are these new packages coming from? It looks like Fluent.Email dependencies, but we're not using it anymore